### ISSUE TITLE
Running `background-test.sh` reveals that `uwunicorn-yt` background download fails

### ISSUE THREAD

**Issue by kohane27** [librephoenix/nixos-config#6]
Hello! Hope you're doing well. Thank you for creating this configuration! I'm switching from Arch to NixOS and this configuration of yours and your videos are exactly what I need to get started! You have my gratitude!

However, I'm still trying to get this installed and notice that the install fails. After reading the faq I notice that the default theme `uwunicorn-yt` stopped working:

```
➜ ./background-test.sh
ashes/ background successfully downloads
atelier-cave/ background successfully downloads
atelier-dune/ background successfully downloads
...
twilight/ background successfully downloads
ubuntu/ background successfully downloads
uwunicorn/ background successfully downloads
uwunicorn-yt/ background download fails
windows-95/ background successfully downloads
woodland/ background successfully downloads
xcode-dusk/ background successfully downloads
```

Not sure how to fix it but I switched theme and trying to install again.

Regardless, thank you again and have a good day!:)

**Comment by obvionaoe** [comment]
Looks like the URL for that background is returning a 404 now
https://github.com/librephoenix/nixos-config/blob/f662ca1e375ac160eba385477b19d1940b2470bb/themes/uwunicorn-yt/backgroundurl.txt#L1
![image](https://github.com/librephoenix/nixos-config/assets/35278323/b07443a0-0a35-4b22-9598-391014134b6c)

**Comment by librephoenix** [comment]
Ah yes..  I love when the background download links become broken XD!  I went and found another background with a different link, so it won't be broken on the default theme!  

Thanks for the heads up!  My system doesn't necessarily tell me it's broken if I already have it downloaded..

I'll push a fix soon once I run a few other tests, since I'm making a few updates to the autoinstall script

**Comment by librephoenix** [comment]
Fixed with a new wallpaper as of [6b66128](https://github.com/librephoenix/nixos-config/commit/6b66128539890c059738f96e97df21ccf4556e5a) :)
